package.json

```diff
@@ -23,11 +23,14 @@
   },
   "scripts": {
     "precommit": "lint-staged",
+    "prepublish": "grunt peerDepInstall",
     "prettier": "prettier --write 'src/**/*.ts' 'tests/**/*.ts'",
     "test": "grunt test"
   },
+  "peerDependencies": {
+    "@dojo/framework": "file:./tarballs/dojo-framework-3.0.0-pre.tgz"
```

Comment:
Why peer dep still?

Reply:
It's only a peer dependency for the time being so this can be made into a tarball and installed itself.